src/WWT.Providers/Providers/GetAuthorThumbnailProvider.cs

```diff
@@ -21,7 +21,7 @@ public override void Run(IWwtContext context)
             }
             string tourcache = ConfigurationManager.AppSettings["WWTTOURCACHE"];
             string localDir = tourcache;
-            string filename = WWTUtil.GetCurrentConfigShare("WWTToursTourFileUNC", true) + String.Format(@"\{0}_AuthorThumb.bin", guid);
+            string filename = ConfigurationManager.AppSettings["WWTToursTourFileUNC"] + String.Format(@"\{0}_AuthorThumb.bin", guid);
```

Comment:
better to do `Path.Combine(ConfigurationManager.AppSettings["..."], $"{guid}_AuthorThumb.bin")`, but not a blocker